### DISCUSSION TITLE
addEventListener

### DISCUSSION THREAD

**Discussion by alex2425** [szimek/signature_pad#578] [Q&A] [answered]
Hello.. somehow 

```
const signaturePad = new SignaturePad(canvas);
signaturePad.addEventListener("beginStroke", () => {
  console.log("Signature started");
}, { once: true });
```

does not work, esepcally the addEventListener to the controll of the signaturepad.. i always get Cannot read properties of undefined (reading 'addEventListener') do i missing something?

**Comment by szimek** [comment, marked as answer]
Hi @alex2425! Are you sure that you're loading the library correctly and that you're loading the latest 4.0.0 version?

Here's a working example: https://codesandbox.io/s/cocky-tdd-4q33j

**Reply by alex2425** [reply]
yep.. sorry my bad i though i had the latest version. It works :)
